src/lib/vrf_lib/standalone.ml

```diff
@@ -1,11 +1,19 @@
+open Module_version
+
 module Context = struct
   type ('message, 'pk) t = {message: 'message; public_key: 'pk}
   [@@deriving sexp]
 end
 
 module Make
     (Impl : Snarky.Snark_intf.S) (Scalar : sig
-        type t [@@deriving eq, sexp, bin_io]
+        module Stable : sig
+          module V1 : sig
+            type t [@@deriving bin_io, sexp]
+          end
+        end
+
+        type t [@@deriving bin_io, sexp, eq]
```

Comment:
this shouldn't have `bin_io` on it

Reply:
Doesn't compile if `bin_io` is missing (same for `sexp`, `eq`)

Reply:
I think this is a bug if that's the case, if we can bin_io the `t` then we aren't version tracking properly